### PR TITLE
EA-194: REST API url should have same standard prefix as other REST e…

### PR DESCRIPTION
**Pull request by mogoodrich**
…ndpoints in OpenMRS

https://openmrs.atlassian.net/browse/EA-194

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/emrapi/web/controller/EmrApiConfigurationController.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 @Controller
-@RequestMapping(value = "/rest/emrapi/configuration")
+@RequestMapping(value = "/rest/**/emrapi/configuration")
```
Comment (mseaton) [inline omod/src/main/java/org/openmrs/module/emrapi/web/controller/EmrApiConfigurationController.java:19]: This hasn't been released, so in theory we could just use `v1`, but this makes sense regardless I think until there is actually a v2+

File: omod/src/main/java/org/openmrs/module/emrapi/web/controller/InpatientVisitsController.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
-@RequestMapping(value = "/rest/emrapi/inpatient")
+@RequestMapping(value = "/rest/**/emrapi/inpatient")
```
Comment (mseaton) [inline omod/src/main/java/org/openmrs/module/emrapi/web/controller/InpatientVisitsController.java:22]: Ditto.  Not really an issue, as I think this controller / endpoint is going to go away once my PR is ready...

**Review by mseaton — APPROVED** [review]
LGTM, thanks @mogoodrich

File: omod/src/main/java/org/openmrs/module/emrapi/web/controller/EmrApiConfigurationController.java (hunk 1 above)
Comment (mogoodrich) [reply omod/src/main/java/org/openmrs/module/emrapi/web/controller/EmrApiConfigurationController.java:19]: Fair enough, I'm just going to merge this in, but feel free to change when you finalize your PR

**Comment by mogoodrich** [comment]
Thanks @mseaton , yeah, feel free to rework the ones your actively working on

**Pull request merged**